### DISCUSSION TITLE
General questions related screen sizes and spacing

### DISCUSSION THREAD

**Discussion by wangzhenux** [twilio-labs/paste#1892] [General] [answered]
1. When we design for Twilio Console with the paste design system (desktop), what (minimum) screen size would you recommend? I took a look at a few teams file, some files use 1200px width, some uses 1440px.
2. When we design with multiple blocks using the paste design system for Twilio Console, what standard spacing would you recommend? For example, in a form, with multiple inputs, the vertical spacing between them should be 24px (same margin as paragraph or heading component) or other spacing sizes (e.g. 20px I saw in some design files as well). I checked out https://paste.twilio.design/tokens#spacings, which is 4px based spacing but it didn’t specify when to use which spacing.

Thank you.

**Comment by loreina** [comment, marked as answer]
Hi @wangzhenux, great questions!

1. Screen sizes are up to you – Paste is fluid as each product may have different width constraints.
2. We don't have spacing guidelines just yet, but are hoping to have them in an upcoming release. We'll follow up here when those are available on the doc site.

**Reply by wangzhenux** [reply]
Thank you for your response, @loreina!

In terms of the screen size, although Paste is fluid, each product should have a minimum supported screen width. As our team is designing for Console, should I ask the Console team about this?

Thank you.

**Reply by loreina** [reply]
Yeah, it'd be great if you could ask Console!
